cli/tests/pcluster/config/utils.py

```diff
@@ -40,28 +40,55 @@ def get_pcluster_config_example():
     return os.path.join(current_dir, "..", "..", "..", "pcluster", "examples", "config")
 
 
-def assert_param_from_file(mocker, section_definition, param_key, param_value, expected_value, expected_message):
+def get_default_values_for_required_clsuter_section_params(cluster_section_dict, only_if_not_present=False):
+    """
+    Provide default values for required parameters for the cluster section.
+
+    This is useful for the cluster section of the config file because, unlike the CFN template,
+    there are no default values defined in mappings.py or in the defaults module used for testing.
+
+    If only_if_not_present is set, then the default values are only added if the
+    cluster_section_dict does not contain keys for the required parameters.
+    """
+    required_cluster_params = [
+        {"config_key": "scheduler", "value": "slurm"},
```

Comment:
config_key --> key

or we can simplify it with: 
```python
required_cluster_params = [{"scheduler": "slurm"}, { "base_os": "alinux2" }]
```

Reply:
Went with the first option.